### ISSUE TITLE
Is there a sample settings and metabox script?

### ISSUE THREAD

**Issue by dougcassidy** [vafour/vafpress-framework#97]
I'm looking for a sample script that shows how to set up the settings, metaboxes, etc.  I'm a little stumped.  I have the framework included, as a library.  Is there a settings array that i  need to filter? do I set up a new framework object?

**Comment by padalec** [comment]
Do you read documentation? http://vafpress.com/documentation/vafpress-framework/getting-started.html

**Comment by dougcassidy** [comment]
Of course.  We have code like
    'fields' => array(
    array(
    'type' => 'toggle',
    'name' => 'tg_1',
    'label' => __('Toggle', 'vp_textdomain'),
    'description' => __('Suits the need to ask user a yes or no option.', 'vp_textdomain'),
    'default' => '1',
    ),
    )
but no clue where to put that code?

**Comment by padalec** [comment]
Do you know how to create theme in WP?

**Comment by dougcassidy** [comment]
Yes.  You do not say in your documentation how to use the the small snippets of code.  I can not find any full example code.  

is it like alchemy?
$custom_metabox = new WPAlchemy_MetaBox(array(
    'id' => '_custom_meta',
    'title' => 'My Custom Meta',
    'template' => STYLESHEETPATH . '/custom/meta.php'
));

**Comment by padalec** [comment]
Look this http://vafpress.com/documentation/vafpress-framework/metabox/builder.html
and try this code in your function.php file:

<pre>
require PATH_TO_VFPRESS.'/vfpress/bootstrap.php';
function my_init_metaboxes()
{
    $meta_box = array(
        array(
            'name'          => 'field_name',
            'type'          => 'textbox',
            'label'         => __('Title'),
        )
    );
    $data = array(
        'id'          => 'uniqe_id',
        'types'       => array('post_types_list'),
        'title'       => __('Title'),
        'priority'    => 'normal',
        'is_dev_mode' => false,
        'template'    => $meta_box,
    );
    $VP = new VP_Metabox($data);
}
// the safest hook to use, since Vafpress Framework may exists in Theme or Plugin
add_action( 'after_setup_theme', 'my_init_metaboxes' );    
</pre>

it was not tested (errors may occur).

Regards

**Comment by dougcassidy** [comment]
Ah, ok, this is closer to what I am looking for.  Thank you.  Although, I did not think to look in the 'builder' sections.  _What I would really like to see are the scripts used to set up your demo site._  If I had those scripts, I could easily build anything from your framework, just by copying your scripts. I have evaluated several frameworks, in the last week or so, and all of them had demo scripts that showed how to use every type of option/metabox/filed.

**Comment by padalec** [comment]
Hi,
this is not my framework I'm just a user who uses it :). Somewhere on github exists demo version  for download or try search this forum http://support.vafpress.com/forum/vafpress-framework/helps/

**Comment by dougcassidy** [comment]
For those searching, this appears to be it https://github.com/vafour/vafpress-framework-theme-boilerplate

**Comment by padalec** [comment]
great this is it! :)

**Comment by Zackio** [comment]
I wish I knew that existed!
